ApeWorX/ape
# feat: logging

### What I did

Opened early because working with @lost-theory on this

fixes: #57 

### How I did it

1.) ^ Creates a logger that is injected into `click` and is able to have its verbosity set via `--verbosity` flag (`click-logging` bulitin), thanks @lost-theory 
2.) Replaces uses of `notify` with `click-logging`.  Meaning: if you did `notify("WARNING, msg)` before, now you do `logger.warning(msg)` and it is virtually the same.
3.) Create a `PluginHelper` class that has the logger as a dependency.  It also has an  `abort()` method. This is injectible into click commands (plugins).

### How to verify it

* Run commands and notice the same output as before
* Try using the `plugin_helper` in a plugin

### Checklist
